pkg/config/piped.go

```diff
@@ -451,7 +453,7 @@ func (r *HelmChartRegistry) Mask() {
 
 type PipedCloudProvider struct {
 	Name string                `json:"name,omitempty"`
-	Type model.ApplicationKind `json:"type,string,omitempty"`
+	Type model.ApplicationKind `json:"type,string"`
```

Comment:
Remove `omitempty` so that Kubernetes-type (`0`) cloud provider will shown in dialog

Reply:
I see 👍 While masking the configuration, I think it would be better if we use the enum string here instead of the enum value (instead of "0", should use KUBERNETES). wdyt?

Reply:
> I think it would be better if we use the enum string here instead of the enum value

I agree, but `PipedCloudProvider.Type` is referenced from so many files so a large modification is needed if we do this.
I made this fix because I thought I should finish fixing the bug first. 
What do you think?